lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -580,6 +580,27 @@ export class UserAgentApplication {
         });
     }
 
+    /**
+     * API interfacing idToken request when applications already have a session/hint acquired by authorization client applications
+     * @param request
+     */
+    ssoSilent(request: AuthenticationParameters): Promise<AuthResponse> {
+        // throw an error on an empty request
+        if (!request) {
+            throw ClientConfigurationError.createEmptyRequestError();
+        }
+
+        // throw an error on no hints passed
+        if (!request.sid && !request.loginHint) {
```

Comment:
@sameerag The original logic here was wrong, since you are checking negative and then throwing, you need `&&`, not `||`. Also added unit tests.

Reply:
Great. Thanks :)